src/common/services/registerUserRequest.ts

```diff
@@ -0,0 +1,22 @@
+import { CreateUserType } from '../../common/types/OperationTypes.types';
+import { useMutation } from '@tanstack/react-query';
+
+const getRequestObject = (values: CreateUserType) => {
+  return {
+    body: JSON.stringify(values),
+    headers: {
+      'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept',
+      'Access-Control-Allow-Origin': '*',
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
+  };
+};
+
+const createUser = async (values: CreateUserType): Promise<any> => {
```

Comment:
`any` is not good type, be more specific. Use type generated from openapi schema

Reply:
you are also getting warning if you run `npm run lint-ts`
```
dorm-social-room-manager/dsrm-frontend/src/common/services/registerUserRequest.ts
  16:60  warning  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

✖ 1 problem (0 errors, 1 warning)
```

Please add this rule to eslintrc.json
```json
"@typescript-eslint/no-explicit-any": "error"
```

Reply:
Changed type to Response.